### ISSUE TITLE
Feature request: Update multiple PDFs

### ISSUE THREAD

**Issue by samboman** [Staubteufel/BulkPDF#26]
I have a number of PDF files which I need to update each file shall have data from one excel-row. Is there a workaround to do this in a smart way?

**Comment by Tadelsucht** [comment]
No, not with this software. You have to use a single template PDF form for BulkPDF.
(But you could try to modify the XML BulkPDF-Configuration dynamically to do what you want)

Or code your wished feature and contribute it to the project.

**Comment by samboman** [comment]
Maybe I go for a way where I split the Excel file into one file per row and dynamically create BulkPDF Configuration dynamically.

**Comment by Tadelsucht** [comment]
Together with BulkPDFConsole a small powershell script should do the trick.
